### PR TITLE
Delay JuMP-dev III submission deadline

### LINKED ISSUE #94: Update JuMP-dev talk submission deadline
To January 4th. Notification January 11th?

@odow @joehuchette

### PR DESCRIPTION
**Pull request by odow**
Closes #94 

@rocarvaj did I muck up the translation?

### CODE REVIEW

File: meetings/santiago2019/index.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ for a flavor of the workshop.
 **To register your attendance at the workshop, (and optionally submit a proposal
 for a talk), [use this form](https://goo.gl/forms/2nEljULHwC3zzcE13).**
 
-Talk proposals are also welcome until December 21st and talk acceptance will
-be notified by January 4th. 
+Talk proposals are welcome until **January 4th** and talk acceptance will be
```
Comment (mlubin) [inline meetings/santiago2019/index.md]: Could you use a strike-through for the old dates to make a clearer that the deadline was extended?

**Comment by rocarvaj** [comment]
I don't know, @odow ... You did some pretty deep changes there 😆. Just kidding, everything looks fine!

File: meetings/santiago2019/index.md [hunk 2]
```diff
@@ -51,8 +51,8 @@ for a flavor of the workshop.
 **To register your attendance at the workshop, (and optionally submit a proposal
 for a talk), [use this form](https://goo.gl/forms/2nEljULHwC3zzcE13).**
 
-Talk proposals are also welcome until December 21st and talk acceptance will
-be notified by January 4th. 
+Talk proposals are welcome until <s>December 21st</s> **January 4th** and talk acceptance will be
```
Comment (mlubin) [inline meetings/santiago2019/index.md]: According to https://en.wikipedia.org/wiki/Strikethrough, `<s>` was deprecated in 1999. Would markdown `~strikethrough~` work?

Comment (odow) [reply meetings/santiago2019/index.md]: `<s>` worked locally haha. Changed to `~~`.

**Pull request merged**